portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,149 @@
+  <head>
+    <meta charset='UTF-8'>
+    <meta name='author' content='Sara Missak'>
+    <title>Contacts</title>
+    
+    <script src='js/script.js'></script>
+
+	<!--Import Google Icon Font-->
+    <link href='https://fonts.googleapis.com/icon?family=Material+Icons' rel='stylesheet'>
+    
+    <!--Import materialize.css from maeralizecss.com-->
+    <link rel='stylesheet' href='css/materialize.min.css'  media='screen,projection'>
+
+    <!--Let browser know website is optimized for mobile-->
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    
+    <!-- Compiled and minified CSS from materalizecss.com-->
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css'>
+
+    <!-- Compiled and minified JavaScript from materalizecss.com-->
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js'></script>
+
+    <!--  Scripts from materalizecss.com-->
+    <script src='https://code.jquery.com/jquery-2.1.1.min.js'></script>
+    <script src='js/materialize.min.js'></script>
+    <script src='js/init.js'></script>
+
+
+    <link rel='stylesheet' href='css/style.css'>
+
+  </head>
+  <body>
+    <!-- Navigation Bar -->
+    <nav class='white'>
+      <div class='nav-wrapper container'>
+        <a id='logo-container' href='index.html' class='brand-logo'>Sara Missak</a>
+        <ul class='right hide-on-med-and-down'>
+          <li><a href='index.html'>About Me</a></li>
+          <li><a href='projects.html'>My Projects</a></li>
+          <li><a class='active'  href='contact.html'>Contact Me</a></li>
+        </ul>
+
+        <!-- For mobile use, edits -->
+        <ul id="nav-mobile" class="sidenav">
+          <li><a class='active' href='index.html'>About Me</a></li>
+          <li><a href='projects.html'>My Projects</a></li>
+          <li><a href='contact.html'>Contact Me</a></li>
+        </ul>
+        <a href="#" data-target="nav-mobile" class="sidenav-trigger"><i class="material-icons">menu</i></a>
+      </div>
+   	</nav>
+
+    <!-- First container with title -->
+	<div id='index-banner' class='parallax-container'>
+      <div class='section no-pad-bot'>
+        <div class='container'>
+          <br><br>
+          <h1 class='header center teal-text accent-3' id='title'>Contact me!</h1>
+          <br><br>
+        </div>
+      </div>
+      <!-- back ground image for first container -->
+      <div class='parallax'><img src='images/background3.JPG' alt='Unsplashed background img 1'></div>
+    </div>
+
+    <!-- Third (White) container -->
+    <div class='container'>
+      <div class='section'>
+        <div class='row'>
+          <div class='col s12 center'>
+        	<h3><i class='mdi-content-send brown-text'></i></h3>
+	        <h4>Send me a message!</h4>
+            <form class="col s12" onsubmit="return validate()" action="mailto:sarammissak@gmail.com" method="post" enctype="text/plain">
```

Comment:
Is there a better way to do this? This way it opens the client's email and they have to manually send the email. I searched it up and it says you can only do it with PHP but I was wondering if you had any other ideas to this?